Formula/csvtk.rb

```diff
@@ -25,6 +26,12 @@ class Csvtk < Formula
 
   def install
     system "go", "build", *std_go_args(ldflags: "-s -w"), "./csvtk"
+    system bin/"csvtk", "genautocomplete", "--shell", "bash", "--file", "csvtk.bash"
+    system bin/"csvtk", "genautocomplete", "--shell", "zsh", "--file", "_csvtk"
+    system bin/"csvtk", "genautocomplete", "--shell", "fish", "--file", "csvtk.fish"
+    bash_completion.install "csvtk.bash" => "csvtk"
+    zsh_completion.install "_csvtk"
+    fish_completion.install "csvtk.fish"
```

Comment:
@Homebrew/core do we have documentation on the new command for this?

Reply:
https://github.com/Homebrew/homebrew-core/blob/6a09d9f78fdaeffce786b0fd50fcb11df2804443/Formula/rclone.rb#L26-L31

I copy it  from rclone. 😄

Reply:
> @Homebrew/core do we have documentation on the new command for this?

https://rubydoc.brew.sh/Formula#generate_completions_from_executable-instance_method

Reply:
```suggestion
    generate_completions_from_executable(bin/"csvtk", "genautocomplete", shell_parameter_format: :arg)
```